docker-run-generate-outputs.sh

```diff
@@ -60,7 +61,8 @@ fi
 
 CMD="pipenv run $PROFILE_CPU_CMD $PROFILE_MEMORY_CMD python -u generate_outputs.py  \
     \"$USER\" /credentials/google-cloud-credentials.json /data/pipeline_configuration.json \
-    /data/raw-data /data/prev-coded /data/output.json /data/output-icr /data/coded \
+    /data/raw-data /data/prev-coded /data/output-message-json-output.json /data/output-individual-json-output.json \
```

Comment:
You're saying "output" and "json" twice here

Reply:
Updated thank you!